modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -336,7 +338,22 @@ var CBIWifiFrequencyValue = form.Value.extend({
 					'VHT20', '20 MHz', htmodelist.VHT20,
 					'VHT40', '40 MHz', htmodelist.VHT40,
 					'VHT80', '80 MHz', htmodelist.VHT80,
+					'VHT80+80', '160 MHz (dual 80 MHz)', htmodelist['VHT80+80'],
```

Comment:
`VHT80+80` looks to be something we could be getting back from iwinfo [(see here)](https://git.openwrt.org/?p=project/iwinfo.git;a=blob;f=iwinfo_lib.c;h=70b080cedfc07be6b001f7084ad63ffe69d92f3f;hb=refs/heads/master#l60).